doc/nrf/config_and_build/configuring_app/advanced_building.rst

```diff
@@ -25,61 +25,34 @@ Compiler options not controlled by the Zephyr build system can be controlled thr
 .. _gs_modifying_build_types:
 .. _modifying_build_types:
 
-Configuring build types
-***********************
+Configuring file suffixes
+*************************
 
-When the ``CONF_FILE`` variable contains a single file and this file name follows the naming pattern :file:`prj_<buildtype>.conf`, then the build type will be inferred to be *<buildtype>*.
-The build type cannot be set explicitly.
-The *<buildtype>* can be any string, but it is common to use ``release`` and ``debug``.
+Zephyr provides the :ref:`zephyr:application-file-suffixes` feature for applications that require a single code base with multiple configurations for different product or build variants (or both).
+When the :makevar:`FILE_SUFFIX` variable is provided, the build system will use Kconfig fragment and devicetree overlay files that match the suffix.
```

Comment:
`When the :makevar:`FILE_SUFFIX` variable is provided, the build system will use Kconfig fragment and devicetree overlay files that match the suffix if they exist.`
it will fall back to those without the suffix if they do not